README.md

```diff
@@ -1,53 +1,57 @@
-
-* running [CusDeb API](https://github.com/tolstoyevsky/cusdeb-api) and [Black Magic](https://github.com/tolstoyevsky/blackmagic);
-* [Node.js](https://nodejs.org/) 12 or above.
-
 ## Configuration
 
 CusDeb Web Client can be configured via the following environment variables (called parameters).
 
-| Parameter        | Description |
-|------------------|-------------|
-| `BM_RPC_URL`     | The URL of the WebSocket API provided by Black Magic. If Black Magic listens on the port 8002, the value of the param *might* look like `ws://127.0.0.1:8002/rpc/token/%token` when running locally. |
-| `CUSDEB_API_URL` | The URL of CusDeb API, including its version. The value of the param *must be* `/api/v1` when running locally. |
-| `HOST`           | The address of the machine Webpack Dev Server is running on. It helps to check CusDeb Web Client on other devices in a local network. |
+| Parameter              | Description                                                                                                                           | Default                                     |
+|------------------------|---------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------|
+| `BLACKMAGIC_URL`       | The URL of the API endpoint provided by the [Black Magic](https://github.com/tolstoyevsky/blackmagic) service.                        | `ws://localhost:8002/bm/token/%token`       |
+| `CUSDEB_ANONYMOUS_URL` | The address of the [CusDeb Anonymous](https://github.com/tolstoyevsky/cusdeb-anonymous) service.                                      | `http://localhost:8007`                     |
+| `CUSDEB_API_URL`       | The address of the [CusDeb API](https://github.com/tolstoyevsky/cusdeb-api) service.                                                  | `http://localhost:8001`                     |
+| `CUSDEB_HELPIK_URL`    | The address of the [CusDeb Helpik](https://github.com/tolstoyevsky/cusdeb-helpik) service.                                            | `http://localhost:8005`                     |
+| `CUSDEB_TZ_URL`        | The address of the [CusDeb Tz](https://github.com/tolstoyevsky/cusdeb-tz) service.                                                    | `http://localhost:8006`                     |
+| `DOMINION_URL`         | The URL of the API endpoint provided by the [Dominion](https://github.com/tolstoyevsky/dominion) service.                             | `ws://localhost:8003/dominion/token/%token` |
+| `HOST`                 | The address of the machine Webpack Dev Server is running on. It helps to check CusDeb Web Client on other devices in a local network. | `0.0.0.0`                                   |
+| `MODE`                 | The webpack configuration option tells webpack to use its built-in optimizations accordingly when it's sets to `development`.         | `development`                               |
```

Comment:
The Webpack configuration option to tell webpack to use its built-in optimizations, depending on the value. It may take the following values: `production`, `development` and `none`.